web/styles/settings.css

```diff
@@ -438,6 +438,7 @@ input[type="checkbox"] {
 .notification-settings-form {
     .subsection-header h3 {
         display: inline-block;
+        margin: 10px 0 5px;
```

Comment:
Why this top margin of 10px is added?

Reply:
I was thinking if we could instead remove that margin but it seems good with the margin. Could have just set `margin-bottom` to 5px as well to not override the margin set by bootstrap, but might be fine since we are in process of removing bootstrap CSS.